FAQ.md

```diff
@@ -242,6 +242,16 @@ generate WebAssembly code from C and C++, and will use LLVM IR in their
 implementation similarly to how PNaCl and Emscripten do today.
 
 
+## Why not use the .net or jvm virtual machines and bytecodes?
```

Comment:
.NET should be all caps.

JVM virtual machines is redundant (java virtual machine virtual machines).  I suggest changing "jvm" to "Java".